docs/_shared/note-dtrain-learn-more.txt

```diff
@@ -0,0 +1,3 @@
+.. note::
+    
+    To learn more about Distributed Training with Determined, visit the :ref:`conceptual overview <multi-gpu-training-concept>` or the :ref:`intro to implementing distributed training <multi-gpu-training-implement>`. 
```

Comment:
It violates my expectation for "Distributed Training" to be capitalized. I think of it as a generic concept rather than a proper noun.

Reply:
you are correct, it should not be capitalized